packages/react-components/src/components/styles/CaptionsSettingsModal.styles.ts

```diff
@@ -86,6 +86,18 @@ export const buttonStyles = (theme: Theme): IButtonStyles => {
   };
 };
 
+/**
+ * @private
+ */
+export const defaultButtonStyles = (): IButtonStyles => {
+  return {
+    root: {
+      borderRadius: _pxToRem(2),
+      margin: _pxToRem(8)
```

Comment:
Do we need to take into account the theme of the styles at all? for when this is used in high contrast mode?

Reply:
Considering Caroline only increased the thickness, this should be applicable to other contrast modes

Reply:
It's looking good in dark theme. Previously made a mistake and applied the primary button stying to this button as well which caused the contrast issue
<img width="111" alt="Screenshot 2025-04-28 at 3 03 22 PM" src="https://github.com/user-attachments/assets/603116e9-683c-480a-ba06-510adf67d337" />